_rules/SC2-4-6-descriptive-headings.md

```diff
@@ -103,27 +110,63 @@ Heading marked up with `role="heading"` that describes the topic or purpose of i
 
 #### Passed example 6
 
-Heading marked up with `h` element that describes the topic or purpose of its section of the content. The heading is visible, but is not included in the Accessibility Tree.
+The heading marked up with `h` element describes the topic or purpose of its section of the content. The heading is visible, but is not included in the Accessibility Tree.
+
+```html
+<h1 aria-hidden="true">Opening Hours</h1>
+<p>We are open Monday through Friday from 10 to 16</p>
+```
+
+#### Passed example 7
+
+The heading marked up with `h` element describes the topic or purpose of a part of its section of the content.
+
+```html
+<h1>Oranges</h1>
+<p>I really like oranges.</p>
+<p>Apples are great too, though.</p>
+```
+
+#### Passed example 8
+
+The heading marked up with `h` element doesn't have a section of content.
+
+```html
+<h1>Oranges on sale</h1>
+```
```

Comment:
For me this would fail our check that all headings are followed by some content, so it feels uncomfortable to have it passing a test to say it describes the topic or purpose of something that isn't there.

Reply:
@ShadowBB, this edit came from you guys. Can you chime in with any arguments on why you feel a heading without content underneath it is not an issue?

Reply:
@EmmaJP, I guess the argument is actually that if the heading markup has been misused to mark something up as a heading though it actually isn't, this would be a failure under succes criterion 1.3.1 Info and Relationships, but not per se under 2.4.6 Headings and Labels.

@kasperisager and I just discussed if there should actually be 2 expectations (per current expectation) that would roughly be something like this:

1. The heading has a section of content
2. The heading is descriptive of its section of content  

... but after thinking about it, I think the first one is actually a 1.3.1 violation.

I suggest keeping the "or doesn't have a section of content" part.

Reply:
Surely then this should be 'not applicable', as there is no content to describe, rather than a pass.

Reply:
Right now the "or doesn't have a section of content" part lives in the expectation, so as the rule is written now, it is a Pass.

If we move this to the Applicability, you will already in the applicability have to judge whether the heading is connected to a section of content or not. 
At least that will limit the possibilities for automating the applicability.

Reply:
Sorry for chiming in to this so late.

The rule should not fail this situation because the rule doesn't fail 1.3.1 and this situation should fail 1.3.1.

This situation should not be not applicable because it is not possible to create an applicability that differentiates between headings with content and headings without content. This can be done in the expectation part of the rule, but can't be done in the applicability part of the rule.

Reply:
Sorry to come back to this after so long. This one is tricky. 

I guess the heading could be of a kind where no content following it would be meaningful. That is not necessarily true of just any heading text.

Something like "Delivery availability" could work with no content below. Perhaps "Oranges on sale" is such a heading, but it isn't obviously so.